src/rule-parse.y

```diff
@@ -199,21 +199,12 @@ rule_attr:
 				{
 				const char *msg = id_to_str($2);
 				if ( ! zeek::util::streq(msg, "<error>") )
-					zeek::reporter->Deprecation(zeek::util::fmt("Remove in v7.1: Using an identifier for msg is deprecated (%s:%d)",
-					                                            current_rule_file, rules_line_number+1));
+					zeek::reporter->Error("Idenfitiers for messages cannot be used for rule events (%s:%d)",
+					                      current_rule_file, rules_line_number+1);
```

Comment:
I'm not sure about the wording here, if anyone has suggestions.

Reply:
We only support event identifiers, so would replace  the whole `else` of `if ( is_event($2) )` with:

```
Error("Identifier %s not an event (%s:%d)", $2, ...);
```

... and not doing the AddAction() anymore below.

Reply:
Removing the check for `<error>` results in a weird error message in one of the tests, so I left that in.

Reply:
This is the result of removing all of it (from signatures.custom-event-errors):

```
   error: Error in signature (./id.sig:19): unknown script-level identifier (non_existing_event)
  error: Identifier <error> is not an event (./id.sig:19)
```